mne/viz/_3d.py

```diff
@@ -594,8 +594,10 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
         the subjects bem and bem/flash folders are searched. Defaults to None.
     seeg : bool
         If True (default), show sEEG electrodes.
-    fnirs : bool
-        If True (default), show fNIRS electrodes.
+    fnirs : str | list | bool | None
+        Can be "channels" or "pairs" to show the fNIRS channel locations or
+        line between source-detector pairs, or a combination like
+        ``('pairs', 'channels')``. True translates to ``('channels')``.
```

Comment:
```suggestion
        ``('pairs', 'channels')``. True translates to ``('channels',)``.
```
Also should we change the default to `('channels', 'pairs')`? If it's the sort of thing people will want most of the time then we should do it.

Reply:
I made it channels by default as that seemed most similar to other modalities. But happy to change to pairs if you prefer. It's explained in the tutorial, so should be accessible to users.

Reply:
What do you think fNIRS people will want to see by default? I wouldn't worry about what the other channels do but rather what's appropriate for your modality...

Reply:
Pairs it is then :) Change committed below.